web/src/pages/CreateRoom/CreateRoom.module.scss

```diff
@@ -0,0 +1,144 @@
+main.createRoom {
```

Comment:
Writing this SAAS made me realize how long it had been since I'd written any! Def feel rusty...